### ISSUE TITLE
automated version bumping

### ISSUE THREAD

**Issue by esuomi** [siili-core/blurhash#16]
Hi! Since this repo has become active again, would you be interested in easy-to-add automated version bumping with git tagging? I'm totally trying to sell my own tool for this here (https://github.com/esuomi/git-revisions), if you're interested I can make a PR and leave some notes on what kind of secrets you need to set to get the things working :)

**Comment by tvirolai** [comment]
Yeah, by all means do! Currently the `pom.xml` provides the info needed for deployment, but the GHA plugin I'm using supports Git tags too.

I'm thinking of migrating the project to a `master`-`develop`-`feature-branch` -model, where merging to master (from develop) automatically creates a release, but completed features are first merged to develop.

**Comment by esuomi** [comment]
Closing this as it's been ages :)
